### ISSUE TITLE
Unable to use CustomTimeouts class in python

### ISSUE THREAD

**Issue by clstokes** [pulumi/pulumi#3815]
The documentation at https://www.pulumi.com/docs/intro/concepts/programming-model/#customtimeouts shows using a `CustomTimeouts` class to provide custom timeout information, but using this with fully-qualified class names, this results in a runtime exception.

#### Code from example
```py
db = Database('db',
    opts=ResourceOptions(custom_timeouts=CustomTimeouts(create='30m')))
```

#### Test Code
```py
import pulumi
from pulumi_gcp import compute

script = "#!/bin/bash\nsudo touch /tmp/a.txt\nsudo yum install -y nginx\nsudo service nginx start"

addr = compute.address.Address("poc")

network = compute.Network("poc")

firewall = compute.Firewall(
    "poc",
    network=network.self_link,
    allows=[
        {
            "protocol": "tcp",
            "ports": ["22"]
        },
        {
            "protocol": "tcp",
            "ports": ["80"]
        }
    ]
)

instance = compute.Instance(
    "poc",
    name="poc",
    machine_type="f1-micro",
    boot_disk={
        "initializeParams": {
            "image": "centos-cloud/centos-7-v20190116"
        }
    },
    network_interfaces=[
        {
            "network": network.id,
            "accessConfigs": [{
                "nat_ip": addr.address
            }]
        }
    ],
    metadata_startup_script=script,
    opts=pulumi.ResourceOptions(custom_timeouts=pulumi.resource.CustomTimeouts(create='30m'))
)
)
```

#### Error
```
Diagnostics:
  pulumi:pulumi:Stack (gcp-instance-nginx-dev):
    error: Program failed with an unhandled exception:
    error: Traceback (most recent call last):
      File "/usr/local/bin/pulumi-language-python-exec", line 85, in <module>
        loop.run_until_complete(coro)
      File "/usr/local/Cellar/python/3.7.6_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/asyncio/base_events.py", line 583, in run_until_complete
        return future.result()
      File "/Users/clstokes/go/src/github.com/pulumi/examples/gcp-py-instance-nginx/venv/lib/python3.7/site-packages/pulumi/runtime/stack.py", line 73, in run_in_stack
        raise RPC_MANAGER.unhandled_exception.with_traceback(RPC_MANAGER.exception_traceback)
      File "/Users/clstokes/go/src/github.com/pulumi/examples/gcp-py-instance-nginx/venv/lib/python3.7/site-packages/pulumi/runtime/rpc_manager.py", line 67, in rpc_wrapper
        result = await rpc_function(*args, **kwargs)
      File "/Users/clstokes/go/src/github.com/pulumi/examples/gcp-py-instance-nginx/venv/lib/python3.7/site-packages/pulumi/runtime/resource.py", line 365, in do_register
        supportsPartialValues=True,
    TypeError: Parameter to MergeFrom() must be instance of same class: expected pulumirpc.RegisterResourceRequest.CustomTimeouts got CustomTimeouts.
    error: an unhandled error occurred: Program exited with non-zero exit code: 1
```

#### Workaround
```py
instance = compute.Instance(
    "poc",
    # ...
    opts=pulumi.ResourceOptions(custom_timeouts={"create": "2m"})
)
```

#### Version Info

Using Python `3.7.6`.

pip:
```
Package    Version
---------- -------
Arpeggio   1.9.2  
attrs      19.3.0 
dill       0.3.1.1
grpcio     1.26.0 
parver     0.2.1  
pip        20.0.2 
protobuf   3.11.2 
pulumi     1.9.0  
pulumi-gcp 2.2.0  
semver     2.9.0  
setuptools 45.1.0 
six        1.14.0 
wheel      0.34.0 
```

**Comment by clstokes** [comment]
Updated original comment with Python version.

```
% python -V
Python 3.7.6
```

**Comment by leezen** [comment]
#3920 fixes this issue. @clstokes can you please confirm the fix works for you and close if so? Thanks!

**Comment by clstokes** [comment]
@leezen What do I need to test this? A specific `pulumi` Python package or do I need an updated CLI too? If just the Python package, which specific one from https://pypi.org/project/pulumi/#history should I use?

**Comment by leezen** [comment]
@clstokes Sorry for the super delayed reply -- not sure how this slipped my filters. This went out with 1.11.0. Though, I actually introduced a regression that prevents using the workaround you have above, which is fixed in 1.11.1.
